### PR TITLE
`cargo-shuttle` README

### LINKED ISSUE #71: Write a README for `cargo-shuttle`
In the `[Deploying](https://docs.rs/shuttle-service/latest/shuttle_service/#deploying)` section there is a link to `cargo shuttle`
which links to `https://docs.rs/crate/cargo-shuttle/latest`. 

That leads to a valid page, but its main content is `cargo-shuttle-0.2.2 is not a library.` which is not particularly welcoming.

### PR DESCRIPTION
**Pull request by nahuakang**
The subcommand sections for `auth`, `status` and `logs` can use some improvements :) 

All feedback is welcome so we can make this a great README for the amazing `cargo-shuttle`.

Closes #71.

### CODE REVIEW

**Comment by nahuakang** [comment]
@chesedo Somehow `Cargo.lock` updated itself again 😓 We can remove those specified versions of `bytes` if needed.

**Review by chesedo — COMMENTED** [review]
These seem perfect, thanks @nahuakang. The `Cargo.lock` changes seem to be fine though.

I'm also getting @ivancernja (our DevRel guy) to review this

File: cargo-shuttle/README.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+```sh
+$ cargo shuttle login --api-key your-api-key-from-browser
+```
+
+### Subcommand: `deploy`
+
+To deploy your shuttle project to the cloud, run:
+
+```sh
+$ $ cargo shuttle deploy
```
Comment (ivancernja) [inline cargo-shuttle/README.md:156]: Double '$' :)

File: cargo-shuttle/README.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+<h1>Development</h1>
+</a>
+
+Thanks for using `cargo-shuttle`! We’re very happy to have you with us!
+
+During our alpha period, API keys are completely free and you can deploy as many services as you want.
+
+Just keep in mind that there may be some kinks that require us to take all deployments down once in a while. In certain circumstances we may also have to delete all the data associated with those deployments.
+
+🚀 To contribute to `cargo-shuttle` or stay updated with our development, please [open an issue, discussion or PR on Github](https://github.com/shuttle-hq/shuttle) and [join our Discord](https://discord.gg/H33rRDTm3p)!
```
Comment (ivancernja) [inline cargo-shuttle/README.md:210]: The rocket emoji looks a bit out of place at the beginning, imo. Can be left as is or be moved to the end of the sentence.

File: cargo-shuttle/README.md (hunk 1 above)
Comment (nahuakang) [reply cargo-shuttle/README.md:156]: I should not submit PRs late in the evening haha :D

**Review by chesedo — APPROVED** [review]

**Comment by chesedo** [comment]
Thanks for this @nahuakang

**Pull request merged**